tests/test_marc.py

```diff
@@ -55,7 +55,8 @@ def test_marc_record_all_fields_transform_correctly():
             "Dewey Call Number 3",
         ],
         citation=(
-            "Célébration : 10 siècles de musique de noël. Language material. "
+            "Célébration : 10 siècles de musique de noël. 2016. "
```

Comment:
I don’t think publication_information works in the citation for the Marc transform but leaving it here for your review

Reply:
Done